src/psrose.c

```diff
@@ -949,6 +976,26 @@ EXTERN_MSC int GMT_psrose (void *V_API, int mode, void *args) {
 		}
 	}
 
+	if (Ctrl->N.active) {	/* Draw best-fitting probability distribution curve */
+		unsigned int k, NP = 361;
+		double a, mu, kappa, pdf, inc = 360.0 / (NP - 1), scl = 2.0 * area * Ctrl->A.inc * D2R;
+		double *xp = gmt_M_memory (GMT, NULL, NP, double);
+		double *yp = gmt_M_memory (GMT, NULL, NP, double);
```

Comment:
It's normally not a good idea to do declaration and assignment in other then last one. VS uses to error on that.
Didn't test it yet though.

Reply:
These are all the same though.  What is not OK (but some compilers allow) is a mix of declarations and actions:

```
double x = 55;
if (p > 5.0) x += 9;
int unused = 99;
...
```

Reply:
It was not that type things that I was referring. Still think we should maintain the code style
```
double *xp, *yp;

xp = gmt_M_memory (GMT, NULL, NP, double);
yp = gmt_M_memory (GMT, NULL, NP, double);
```
But as is, VS2019 accepts it.

Reply:
Sure, and *.c is full of these things - has been for years.  Valid syntax in C.